plume-common/src/activity_pub/mod.rs

```diff
@@ -128,6 +128,15 @@ pub enum SignatureValidity {
     Absent,
 }
 
+impl SignatureValidity {
+    pub fn is_secure(&self) -> bool {
+        match self {
+            SignatureValidity::Valid => true,
+            _ => false,
```

Comment:
Why not `self == SignatureValidity::Valid`?

Reply:
Because `PartialEq` is not implemented for SignatureValidity? Joking, it can be derived. I just copied the whole match from [src/instance.rs](https://github.com/Plume-org/Plume/blob/0a5d435249f9f4b4df7f42fc40578af302a91954/src/routes/instance.rs#L218), moved the todo somewhere-else, and basically forgot there were an shorter and cleaner way